src/Umbraco.Cms.Persistence.Sqlite/Services/SqliteDistributedLockingMechanism.cs

```diff
@@ -83,7 +82,7 @@ private class SqliteDistributedLock : IDistributedLock
                         throw new ArgumentOutOfRangeException(nameof(lockType), lockType, @"Unsupported lockType");
                 }
             }
-            catch (SqlException ex) when (ex.Number == 1222)
+            catch (SqliteException ex) when (ex.SqliteErrorCode == SQLitePCL.raw.SQLITE_BUSY)
```

Comment:
[SQL error 1222](https://docs.microsoft.com/en-us/sql/relational-databases/errors-events/mssqlserver-1222-database-engine-error?view=sql-server-ver16) occurs when the lock request timeout period exceeded, but this code is specific to SQLite. If I'm correct, the similar exception from SQLite will have [error code 5 (SQLITE_BUSY)](https://www.sqlite.org/rescode.html#busy). This should be double-checked though!